include/ardpolyfills/arduino.h

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Arduino.h
+ *  Copyright 2020 ItJustWorkTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#ifndef Arduino_h
+#define Arduino_h
+#include <cctype>
+#include <cmath>
+#include <cstdlib>
+
+// Arduino Math functions
+using std::abs;
+long constrain(long, long, long);
```

Comment:
Chen brought to my attention that none of the arguments are actually named, while it is valid its a bit odd because it signifies that they are unused.
Would be good to properly name them so that they are insync with the implementation.

Reply:
+1